Duplicati/Library/Backend/WEBDAV/WEBDAV.cs

```diff
@@ -59,6 +84,16 @@ public class WEBDAV : IBackend, IStreamingBackend
         //private static readonly byte[] PROPFIND_BODY = System.Text.Encoding.UTF8.GetBytes("<?xml version=\"1.0\"?><D:propfind xmlns:D=\"DAV:\"><D:allprop/></D:propfind>");
         private static readonly byte[] PROPFIND_BODY = new byte[0];
 
+        /// <summary>
+        /// The default timeout in seconds for PUT/GET file operations
+        /// </summary>
+        private const int LONG_OPERATION_TIMEOUT_SECONDS = 30000;

```

Comment:
I like this distinction between long/short. Should it be a general option?

Reply:
I believe so, because for operations such as fetching lists, or deleting, it is expected to be much shorter than a 50mb+/- block upload.

I am wondering if they should be constants or a parameter with a set default, allowing the user to override via configuration if they deem necessary.